firefox-ios/Client/Frontend/TrackingProtection/TrackingProtectionModel.swift

```diff
@@ -136,6 +136,13 @@ class TrackingProtectionModel {
         }
     }
 
+    func getCertificatesViewModel() -> CertificatesViewModel {
```

Comment:
Do we ever need multiple instances of the CertificatesViewModel? If no, this can simply be a lazy var `certificatesViewModel`.

Reply:
also instead of view models, we should use redux. I would like to see a TODO ticket if we want to do this in a separate PR. This logic seems like it should live in a certificate manager that lives in the middleware

Reply:
@ih-codes yes, as I know this may used in future.
@cyndichin I am going to add a TODO